maro/cli/k8s/executors/k8s_aks_executor.py

```diff
@@ -462,7 +464,7 @@ def pull_data(self, local_dir: str, remote_path: str):
         abs_target_dir = get_reformatted_target_dir(abs_local_dir)
         os.makedirs(abs_target_dir, exist_ok=True)
         if not source_path.startswith("/"):
-            raise CliException("Invalid remote path")
+            raise FileOperationError(f"Invalid remote path: {source_path}\nShould be started with '/'.")
```

Comment:
Should be started with '/' -> must be absolute path.

Reply:
No actually, this is not an absolute path, this is a "relative path" that needs to follow the rules we use here